Extension/CHANGELOG.md

```diff
@@ -1,32 +1,24 @@
 # C/C++ for Visual Studio Code Changelog
 
-## Version 1.25.2: April 25, 2025
-### Bug Fixes
-* Fix a crash in `read_double`. [#13435](https://github.com/Microsoft/vscode-cpptools/issues/13435)
-* Fix a crash with Copilot hover.
-
-## Version 1.25.1: April 22, 2025
+## Version 1.25.3; April 28, 2025
```

Comment:
I think we usually use a colon instead of semi-colon here

Reply:
fixed